tests/vcs/git/GitCommitBearTest.py

```diff
@@ -267,6 +269,108 @@ def test_body_checks(self):
                          [])
         self.assertTrue(self.msg_queue.empty())
 
+    def test_check_issue(self):
+        # Keyword Matching with regexp
+        self.git_commit('Shortlog\n\n'
+                        'First line, blablablablablabla.\n'
+                        'Another line, blablablablablabla.\n'
+                        'Fix #123')
+        self.assertEqual(self.run_uut(issue_checks=True,
+                                      keyword_regex='Fix\s+#[0-9]+\s*'), [])
+
+        # Adding BitBucket remote for testing
+        self.run_git_command('remote', 'add', 'test',
+                             'https://bitbucket.com/user/repo.git')
+
+        self.git_commit('Shortlog\n\n'
+                        'First line, blablablablablabla.\n'
+                        'Another line, blablablablablabla.\n'
+                        'Closes #123')
+        self.assertEqual(self.run_uut(issue_checks=True,), [])
+
+        # Keyword Matching with regexp, no match found
+        self.git_commit('Shortlog\n\n'
+                        'First line, blablablablablabla.\n'
+                        'Another line, blablablablablabla.\n'
+                        'Closes #123')
+        self.assertEqual(self.run_uut(issue_checks=True,
+                                      keyword_regex='Fix\s+#[0-9]+\s*'),
+                         ['No match found for the regular '
+                          'expression provided: Fix\s+#[0-9]+\s*'])
+
```

Comment:
[1-9] ? but that disables 0 even in numbers like 70.

Although anyway that was just a test, right?
It depends on the user to correctly write the regex.